app/Http/Controllers/UserController.php

```diff
@@ -62,8 +61,9 @@ public function createPDF(){
         $users = User::all();
         $html = view('docs.userPdf', ['users' => $users , 'title' => 'User Mangement']);
         $document->WriteHTML($html);
-        Storage::put('pdfs/user_information.pdf', $document->Output('user_information.pdf', "S"));
-        $document->Output('user_information.pdf', "I");
+        date_default_timezone_set('America/New_York');
```

Comment:
timzone will be configured on the application you never need to customize here. please remove this line. and explain why you setup the timezone here